vowpalwabbit/CMakeLists.txt

```diff
@@ -295,7 +299,9 @@ endif()
 target_include_directories(vw PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
-  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+  ${FLATBUFFERS_INCLUDE_DIR}
```

Comment:
This should be brought in with a flatbuffers target not like this

Reply:
The target created `fbschemas` is a custom target and afaik you can't use `target_include_directories` on custom targets. However the flatbuffers include dir needs to be added to vw. Is there a better way?

Reply:
`FLATBUFFERS_INCLUDE_DIR` comes from 
`find_package(flatbuffers REQUIRED)` right?

If it is header only we should do something similar to what we do for RapidJson.

Reply:
Yup I see, addressed [here](https://github.com/VowpalWabbit/vowpal_wabbit/pull/2496/commits/d2952344eda31d3ee5f5275d64aa04670c83e2ad)